src/plugins/intel_gpu/src/kernel_selector/core/actual_kernels/roi_align/roi_align_kernel_ref.cpp

```diff
@@ -2,6 +2,7 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 #include "roi_align_kernel_ref.h"
+
 #include <kernel_selector_utils.h>
 
 namespace kernel_selector {
```

Comment:
in line 18 we need to add blocked layouts

Reply:
Done